doc/UsersGuide/source/omjulia.rst

```diff
@@ -209,12 +214,14 @@ Similary we can also use setInputs() to set a value for the inputs during variou
 
 >>> setInputs(mod,"cAi=1")
 
-And then finally we can simulate the model using, The simulate() API can be used in two methods
+And then finally we can simulate the model using, The simulate() API can be used in following methods
   1) without any arguments
-  2) resultfile names provided by user (only filename is allowed and not the location)
+  2) resultfile (keyword argument) - (only filename is allowed and not the location)
```

Comment:
Use the sphinx numbering instead of manual numbering.

Reply:
fixed the numbering

Reply:
Squash the commits to one.